pkg/api/apikey.go

```diff
@@ -20,6 +20,11 @@ import (
 //
 // Will return auth keys.
 //
+// Deprecated: true.
+//
+// Deprecated please use GET /api/serviceaccounts and GET /api/serviceaccounts/{id}/tokens instead
```

Comment:
super nit: I'd add a punctuation mark after `deprecated`. Something like `Deprecated: please use...` or `Deprecated. Please use...`.

Reply:
i do not think it's nit. this is about clear communication on this. and should be reviewed as such